stubs/gdb/METADATA.toml

```diff
@@ -0,0 +1,5 @@
+version = "12.1.*"
+extra_description = """\
+  Type hints for GDB's \
+  [Python API](https://sourceware.org/gdb/onlinedocs/gdb/Python-API.html).\
+"""
```

Comment:
You can add the following line to skip running stubtest:

```python
stubtest = false
```

Reply:
I’ll try that, thanks!